.config/dotnet-tools.json

```diff
@@ -0,0 +1,12 @@
+{
+  "version": 1,
+  "isRoot": true,
+  "tools": {
+    "bundlerminifier.core.tool": {
```

Comment:
ignore my earlier comment about the bundleConfig file -- i see this is how it's all tied together